### PR TITLE
OTC-811: Error messages added for insuree code validation.

### PR DESCRIPTION
**Pull request by wzglinieckisoldevelo**
Ticket: https://openimis.atlassian.net/browse/OTC-811

Required dependency:
https://github.com/openimis/openimis-be-core_py/pull/148

Changes:
- Validation query now returns code and error message in addition to a true/false value.
- Query now checks whether user has required permissions.

### CODE REVIEW

**Comment by olewandowski1** [comment]
@dborowiecki @wzglinieckisoldevelo I'm wondering about this _isValid_ change, should we totally base on codes now and get rid of _isValid_ state? The whole code support must be implemeneted on FE, why can't we stick to the _isValid_ state and get its status from BE?

**Comment by dragos-dobre** [comment]
@wzglinieckisoldevelo What's the status of this PR? If there is still work to do please move the ticket back to to do.

**Comment by wzglinieckisoldevelo** [comment]
@dragos-dobre 
I was waiting for dependency to be merged, because tests were failing. I wanted to move PR to open status once this other repository is merged, so I can be sure that everything works as intended.

**Review by malinowskikam — APPROVED** [review]

**Review by delcroip — APPROVED** [review]

**Pull request merged**
